scripts/standalone_embed.sh

```diff
@@ -41,6 +41,23 @@ EOF
         --health-retries=3 \
         milvusdb/milvus:v2.3.7 \
         milvus run standalone  1> /dev/null
+}
+
+start() {
+    res=`sudo docker ps|grep milvus-standalone|grep healthy|wc -l`
+    if [ $res -eq 1 ]
+    then
+        echo "Milvus is running."
+        exit 0
+    fi
+
+    res=`sudo docker ps -a|grep milvus-standalone|wc -l`
+    if [ $res -eq 1 ]
+    then
+        sudo docker start milvus-standalone 1> /dev/null
```

Comment:
We'd better extract the wait_for_milvus_running as a utility helper function. Add it to `docker start` too.

Reply:
changed